### ISSUE TITLE
CC2650STK: Disconnect from 6lbr occasionally

### ISSUE THREAD

**Issue by naivewong** [cetic/6lbr#311]
I used raspberry pi with rasbian 4.1 running 6lbr whose version was 1.3.3. And i used nullRDC on cc2650stk. At fist, i can access the [bbbb::100]/sensors.html and the graph of the two sensortags is perfect. But after a while when i clicked the "web", randomly one sensortag may be disconnected and it would not reconnect. Just like the screenshot below. And then i switched to Beaglebone whose 6lbr version was also 1.3.3 and the same disconnection problem occurred.
![problem5](https://user-images.githubusercontent.com/26733153/27325732-ee4833ee-55db-11e7-9a95-cf511390c03a.JPG)

**Comment by laurentderu** [comment]
I believe this is a bug solved in the later 1.4.x versions. Could you check with version 1.4.1 or one of the dev snapshots ?

**Comment by naivewong** [comment]
@laurentderu I tried this morning. And I found that at first I can access the "web" of two sensors, but after a while, one of the sensors' "web" can not be accessed. But strangely the connection showed on the webpage of 6lbr is complete. Just like this two pics.
![6lbr1 5_2](https://user-images.githubusercontent.com/26733153/27506120-7c08783c-58e4-11e7-9b28-25c10c2cb2bf.JPG)
![6lbr1 5_3](https://user-images.githubusercontent.com/26733153/27506121-7eeb9368-58e4-11e7-9e0c-bc5568fd8f51.JPG)
